woodwork/type_sys/inference_functions.py

```diff
@@ -29,6 +29,12 @@ def categorical_func(series):
 
 
 def integer_func(series):
+    if integer_nullable_func(series) and series.notnull().all():
```

Comment:
Can we verify that `notnull().all()` is the fastest method we can use? We can use something like this:
- https://stackoverflow.com/a/29530601/2512385